instrumentation/sequel/test/opentelemetry/instrumentation/sequel_test.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Instrumentation::Sequel do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Sequel::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:memory_database) { Sequel.connect('sqlite::memory:') }
+  let(:db_file) { Tempfile.new }
+  let(:file_database) { Sequel.connect("sqlite://#{db_file.path}") }
+  let(:model) { Class.new(Sequel::Model(:documents)) }
+
+  before do
+    instrumentation.install
+    memory_database.extension :opentelemetry
+    memory_database.create_table!(:documents) do
+      primary_key :id
+      String :title
+    end
+    file_database.extension :opentelemetry
+    model # Instantiate the model so that it doesn't interfere with further calls
+
+    exporter.reset
+  end
+
+  describe 'database methods' do
+    it 'before query' do
+      _(exporter.finished_spans.size).must_equal 0
```

Comment:
Please use predicate assertions.

These will be more explicit and print the contents of the `finished_spans` in the error message as opposed to the cryptic `expected 0 but was 1`

```suggestion
      _(exporter.finished_spans).must_be_empty
```